src/Symfony/Bridge/Doctrine/Middleware/AutoClose/AutoCloseDriver.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Middleware\AutoClose;
```

Comment:
What about "IdleConnection" instead of "AutoClose" everywhere? (sorry, naming things :) )

Reply:
"GoneConnection", "LostConnection"?

The problem isn't that the connection is idle, but that the server closes the connection without notifying the client.
By the way, we'll have some docs to write because theoretically, this shouldn't happen if the client timeout is configured to a shorter value than the server timeout (unfortunately, this is rarely the case so this PR is definitely needed).

Reply:
the only thing we monitor is the idle time, so "idle" looks appropriate to me